DANE_s_JSONem/.gitignore.txt

```diff
@@ -44,30 +44,71 @@ out/**
 *.ipr
 *.iws
 
-##############################
-## Eclipse
-##############################
-.settings/
-bin/
-tmp/
-.metadata
-.classpath
-.project
-*.tmp
-*.bak
-*.swp
-*~.nib
-local.properties
-.loadpath
-.factorypath
+# User-specific stuff
+.idea/**/workspace.xml
+.idea/**/tasks.xml
+.idea/**/usage.statistics.xml
+.idea/**/dictionaries
+.idea/**/shelf
 
-##############################
-## NetBeans
-##############################
-nbproject/private/
-build/
-nbbuild/
-dist/
-nbdist/
-nbactions.xml
-nb-configuration.xml
+# Generated files
+.idea/**/contentModel.xml
```

Comment:
jo vypadá to ok, ale ještě bych tam přidal konfiguraci pro javu -> aby tam nebyla složka /out